ruby-challenges.rb

```diff
@@ -33,4 +54,40 @@
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
 
-# Pseudo code:
\ No newline at end of file
+# Pseudo code:
+# create fxn that takes in any given str and checks if it is a palindrome
+# if statement for string == string.reverse to see if letters are same back and forth
+# str interpol for true and false of string == string.reverse 
+
+# first try 
+def palindrome? (string)
+      if string == string.reverse
+          "#{string} is a palindrome"
+      else
+          "#{string} is not a palindrome" 
+      end
+  end
+   
+  # p palindrome?(palindrome_test_case1)
+  # p palindrome?('Racecar')
+
+  # the fxn should work in theory but isnt working for palindromes??
+
+#  im dumb. i console logged both and now see that it isnt the same because Caps dont == small caps D: 
+# TRY NUMBER 2! 
+# Pseudo code:
+# create fxn that takes in any given str and checks if it is a palindrome
+# convert string to all lowercase
+# i used gsub to make my fxn more dynamic.
+# if statement for string == string.reverse to see if letters are same back and forth
+# str interpol for true and false of string == string.reverse 
+
+def palindrome? (string)
+  small_case_string = string.downcase.gsub(/\s+/, "")
+  if small_case_string == small_case_string.reverse
+      "#{small_case_string} is a palindrome"
+  else
+      "#{small_case_string} is not a palindrome" 
+  end
+end
```

Comment:
You don't have to use gsub... You can just compare string.downcase to string.downcase.reverse( or use upcase instead of downcase).  Nice find on the gsub.  you can use that in other things!